### ISSUE TITLE
Official badge or button to show accessibility commitment

### ISSUE THREAD

**Issue by solonovamax** [w3c/wai-statements#204]
Similar in purpose to the [Accessibility Statement](https://www.w3.org/WAI/planning/statements/), it would be nice to have an "Accessibility Badge" of some sorts that could be added to the footer of a website.

I run a personal website, where I do not wish to go full corpo-speak. However I *do* wish to show my commitment to accessibility & that I care about accessibility.

For this purpose, it would be nice to have some kind of SVG badge/button that could be easily added to the footer of my website, possibly with a link to a dedicated page, showing my commitment to accessibility.

Note: I am not using "button" in the sense of "html `<button>` element", rather I am using it in the same way one would use it for an 88x31 button.

Here are a couple of ideas for elements the badge/button might include (it could include all, or just some, of them):
- the W3C logo
  - is there a logo/icon for the WCAG or the WAI? if yes, those should be used instead.
  - maybe some sort of dedicated "accessibility logo" might be better? idk
- the text "WCAG Level" or "Accessibility Level"
  - possibly there could be a variant of the badge/button which says "Accessible by Default" to showcase one's full commitment to accessibility.
- the level to which the website conforms
  - the intent of this would be to include the level which the website *mostly* conforms to, not the one to which it fully conforms. if the button/badge links to another page, then that other page would outline any exceptions. the goal of the button would be to outline the broad strokes of conformance level.

Ideally, a couple different styles of badge could exist, so that anyone using the badge can pick the style which matches their website theme the best.

Here is a hypothetical example of how the badge could look (of course, I don't think this is actually what should be used, there are multiple changes I would make do it, this is just an example):
<img src="https://img.shields.io/badge/WCAG%20Level-AAA-blue?style=for-the-badge&logo=data%3Aimage%2Fsvg%2Bxml%3Bbase64%2CPHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI0NiIgaGVpZ2h0PSIyNi41NzQiIHZpZXdCb3g9IjAgMCAzNjAgMjA3Ljk3Ij48ZyBmaWxsPSIjZmZmIj48cGF0aCBkPSJNMzE0LjA5IDE1NC4yMTZjLTYuODQ5LTUuNDM2LTE2Ljg0OS00LjI4LTIyLjI4NiAyLjU2OS02LjczNCA4LjQ5LTEyLjg1IDEzLjYtMTkuODIzIDE2LjU3Ni0xMi44NTkgNS40NzEtMjMuNzYgMy4xNTktMzQuMjk4LTcuMjgzLTEwLjMyNy0xMC4yMjgtMjAuMzg5LTIxLjM3NS0zMC4xNS0zMi4xNzktNC43NzUtNS4yODctOS41NTktMTAuNTgyLTE0LjQwNC0xNS44MTUtOC4yOC04Ljk0Mi0xNi42MDItMTcuODQ2LTI0LjkxNi0yNi43NjJsLS43MzMtLjc4NWMtMTguMDg2LTE5LjM1Ni0zNi43NzgtMzkuMzczLTU0LjgxLTU5LjQ3OUM5OC4zNzEgMTUuMTEgNzYuMjggNy4yMTEgNTUuMDI2IDEwLjQwNSAzMy4zMDUgMTMuNjUzIDE1Ljc0MiAyNy41NTUgNS41NjUgNDkuNTRjLTkuODA2IDIxLjE3NC02LjQ4NyA0NS4yMzQgOC42NSA2Mi43ODggMjYuNjI4IDMwLjg4MiA0OS4wOSA1Ni40MTYgNzUuNCA4MC40MzIgMTAuNzYgOS44MTQgMjMuNzc5IDE1LjAzOSAzNy42NDQgMTUuMTEgOC45OTQuMDQ1IDIyLjU1LS45NyAzNi4zODItMTAuMjgyIDYuMDU1LTQuMDc3IDExLjY5NC04LjI3IDE0LjE3NC0xNC42NyAyLjUyNS02LjU0OC4xNjctMTQuMDc2LTUuNzQ2LTE4LjMyMi01LjczNy00LjEzLTEzLjI2NC0zLjg0LTE4LjczNy43MDYtLjY5LjU3My0xLjQzOSAxLjIzNi0yLjI2IDEuOTUtOC41OTcgNy41MjktMTkuMjk0IDE2LjkwMy00MS4xMDQgMS4yNDUtMTkuOTgtMTQuMzUyLTM1LjQ1My0zMy4xNi01My4zOC01NC45NDEtNS44NDItNy4wOTctMTEuODg5LTE0LjQ0LTE4LjM1LTIxLjk0Mi02LjkwMS04LjAwNi04LjM5Mi0xOS4wMi0zLjg5MS0yOC43NTYgMy45MjctOC40NzIgMTEuNTEtMTkuMDExIDI1LjM4NC0yMS4wODQgMTAuNTExLTEuNTggMjIuMjk0IDIuNjIgMjkuMzI5IDEwLjQ1OCAxOC4xODEgMjAuMjgyIDM3LjAxNyA0MC40NTggNTUuMjMzIDU5Ljk1NGE4MzA4Ljc3NiA4MzA4LjY5NiAwIDAgMSAyNS41NjIgMjcuNDVjNC43NTYgNS4xMzcgOS40NTIgMTAuMzM1IDE0LjIxNyAxNS42MTMgMTAuMDEgMTEuMDc3IDIwLjM1NCAyMi41MjQgMzEuMjgxIDMzLjM1NCAxOS43MjYgMTkuNTQ5IDQ0LjI0NSAyNC41IDY5LjA3MyAxMy45MzYgMTEuOTI0LTUuMDkzIDIyLjE3LTEzLjM2MyAzMi4yMzMtMjYuMDQ2IDUuNDI4LTYuODQgNC4yOC0xNi44MzEtMi41NjktMjIuMjc3Ii8%2BPHBhdGggZD0iTTMzOC45MDggNjYuMTFjMTAuMjU2LTQuODYzIDE4LjMyMy0xNC4wMTYgMTkuMDkyLTI1LjEyIDEuOTUtMTkuMjc0LTEzLjkyLTM3LjcyLTM3LjI2NS00MC41NzEtMTQuMzMzLTEuODE4LTI3LjYzMyAyLjQyNy0zNy4wNiAxMC4yMzgtNS4wNCA0LjE5My02LjQgMTEuOTUtMi4wMyAxNy40MjMgNC43NzQgNS42OTIgMTAuMjgyIDYuNjkgMTkuMjkzIDEuNzM5IDMuMjMtMS44MzcgOC4wODYtMy4zNjMgMTMuNDI1LTIuOTkyIDEwLjA3LjY5NyAxMi45NTcgOS43NyAxMi42NzQgMTMuODkxLS4wNDQuNjEuMDE4IDE0LjY2LTE0LjE2NSAxMy42ODFsLTUuMzUtLjM3Yy01Ljk1Ni0uNDA2LTExLjkxNCA1LjE2Mi0xMi4zNTYgMTEuNTM0LS40NzcgNi43OCA0LjQ4NCAxMi43MDEgMTAuNjUzIDEzLjEzNGw2LjU3Ni40NUMzMzAuNDkgODAuNCAzMjguNiA5NS43NTYgMzI4LjIyIDk4LjIxMWMtLjM1NCA3LjA2LTExLjY4NiAxOS4wMzctMzEuODk4IDUuNDE4LTYuODU3LTQuNjE2LTU1Ljc3Mi02Ni4wNTMtNTUuNzcyLTY2LjA1My0yMy4xNzctMjkuOTY2LTYyLjU2LTM1LjI0My05My42MzYtMTIuNTQyYTE1Ljc3IDE1Ljc3IDAgMCAwLTYuMzIgMTAuMzcgMTUuOCAxNS44IDAgMCAwIDIuODcgMTEuNzkxIDE1Ljc3IDE1Ljc3IDAgMCAwIDEwLjM2MiA2LjMyIDE1LjczMyAxNS43MzIgMCAwIDAgMTEuOC0yLjg2OGMyMS4yLTE1LjQ5IDQwLjA5Ni02LjI1OCA0OS44MTQgNi4zMSAwIDAgNTIuNTE2IDY3LjU0NiA2Ni4wMzcgNzQuNzA0IDguNzczIDQuNjQzIDE3LjkzNCA3LjMyNiAyNy4yNDYgNy45NyAyOC41OCAxLjk3NiA0NS45ODQtMTMuNzY4IDUwLjY4LTM1LjM0IDMuMDgtMTQuMTQtNi4wNjQtMzEuODA4LTIwLjQ5NS0zOC4xOCIvPjwvZz48L3N2Zz4%3D" width="253.875" height="42">

Such a badge/button might also be useful for non-personal websites, as it would make a website's commitment to accessibility much more visible & help advertise that the website in question cares very much about it.
I have seen a few websites which just have the text "Accessibility" in their footers, however I think having a badge/button would be much nicer & stand out better.

if such a badge already exists, then I could not find it.
